benchmark/run_phylophlan.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+# usage: run PhyloPhlAn to build phylogenetic tree of whole genomes
+
+# prerequisites:
+# one needs to install the development version of PhyloPhlAn:
+#   hg clone https://bitbucket.org/nsegata/phylophlan
+#   cd phylophlan
+#   hg up dev
+# one needs the follow programs installed and callable from $PATH:
```

Comment:
This is the development version Nicola and Francesco are working on. After talking to them they showed me these commands. They said depite not having a version number, this version is pretty stable. They expect to resume the development from April to add some major improvements.